scripts/wrappers/common/utils.py

```diff
@@ -146,6 +146,35 @@ def get_dqlite_info():
     return info
 
 
+def get_etcd_info():
+    kube_apiserver_args = os.path.expandvars("${SNAP_DATA}/args/kube-apiserver")
+    with open(kube_apiserver_args, "r") as f:
+        kube_apiserver_args_content = f.read()
+        etcd_endpoints = []
+        for line in kube_apiserver_args_content.split("\n"):
+            if "etcd-servers" in line:
+                # list all etcd endpointss
+                for endpoint in line.split("=")[1].split(","):
```

Comment:
this will cause the `get_etcd_info` to fail if any single line does not have `=`.

Reply:
Added a check for this.